src/main/scala/net/cakesolutions/CakePlatformPlugin.scala

```diff
@@ -19,30 +25,33 @@ object CakePlatformKeys {
    * We have strict conflict resolution, so it'll catch any mismatches
    * and force you to update your build.sbt
    */
-  object versions {
-    val akka = "2.4.17"
-    val play = PlayVersion.current
+  object Versions {
+    val akka: String = "2.4.17"
+    val play: String = PlayVersion.current
   }
 
-  /** Convenient bundles for depending on platform / core libraries */
-  object deps {
+  /** Convenient bundles for depending on platform/core libraries */
+  object Dependencies {
     // WORKAROUND https://issues.apache.org/jira/browse/CASSANDRA-10984
     //            cassandra needs a subset of netty 4
-    val Cassandra = {
```

Comment:
upper case naming for a constant value was intentional. If you use lower case you risk the very real possibility of import collisions with package names.